pantheon-files-daemon/FileManager1.vala

```diff
@@ -43,7 +43,7 @@ public class FileManager1 : Object {
         string cmd = "pantheon-files -t";
 
         foreach (string s in uris) {
-            cmd += (" " + s);
+            cmd += (" \"" + PF.FileUtils.sanitize_path (s) + "\"");
```

Comment:
Yes, I agree about the single quotes - it just sounded you were questioning the whole approach.